notebooks/cva_opt_with_channels_example.ipynb

```diff
@@ -0,0 +1,435 @@
+   "metadata": {},
+   "source": [
+    "![](../images/TQ42_Banner.png)\n",
+    "\n",
+    "\n",
+    "# Welcome to TQ42!\n",
+    "\n",
+    "## Example how to use channels with CVA_OPT\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "dcd8d3ef-3069-4489-8397-8cba3b3ee769",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from tq42.client import TQ42Client\n",
+    "from tq42.experiment_run import ExperimentRun\n",
+    "from tq42.organization import list_all as list_all_organizations\n",
+    "from tq42.project import list_all as list_all_projects\n",
+    "from tq42.experiment import list_all as list_all_experiments\n",
+    "from tq42.experiment_run import ExperimentRunStatusProto\n",
+    "from tq42.algorithm import AlgorithmProto\n",
+    "from tq42.compute import HardwareProto\n",
+    "import json\n",
+    "import pandas\n",
+    "import asyncio\n",
+    "import numpy as np\n",
+    "from tq42.channel import Channel, Ask, Tell\n",
+    "import OptimizationTestFunctions as otf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c9ed1ab2-279f-4079-8719-c2f2fd18847b",
+   "metadata": {},
+   "source": [
+    "### Get your experiment id by descending the organization, project, experiment hierarchy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "8c4d62de633fd4c1",
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"device_code\":\"v-YmkmEoO3Hny9AkO9DZ-eTg\",\"user_code\":\"CRWR-ZNTN\",\"verification_uri\":\"https://auth.terraquantum.io/activate\",\"expires_in\":900,\"interval\":5,\"verification_uri_complete\":\"https://auth.terraquantum.io/activate?user_code=CRWR-ZNTN\"}\n",
+      "If a browser does not open, please access this URL: https://auth.terraquantum.io/activate?user_code=CRWR-ZNTN to login. Also check the code: CRWR-ZNTN\n",
+      "Authentication is successful, access token is saved in: keyring.\n",
+      "org:4ae7d0af-80f3-4272-a9b2-c3e17c3594b9\n",
+      "proj:c03af6c1-fe5e-45fb-95c1-8f61e9af5b12\n"
+     ]
+    }
+   ],
+   "source": [
+    "tq42client = TQ42Client()\n",
+    "tq42client.login()\n",
+    "org_list = list_all_organizations(tq42client)\n",
+    "project_list = list_all_projects(client=tq42client, organization_id=org_list[0].id)\n",
+    "experiment_list = list_all_experiments(client=tq42client, project_id=project_list[0].id)\n",
+    "experiment_id = experiment_list[0].id"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "5bbbd0d2-4863-459b-a25f-bf4424fc94d3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Running experiment within: Org 4ae7d0af-80f3-4272-a9b2-c3e17c3594b9, Proj 81bf18de-5a56-4e7c-8534-6ea46402f90e and Exp 1c921f32-f425-43cf-8318-4f3bf6a3a9ce`\n"
```

Comment:
not necessarily important but disclosing your ids with this notebook is not the best practice.